website/docs/docs/deploy/continuous-integration.md

```diff
@@ -27,6 +27,27 @@ When the CI run completes, you can view the run status directly from within the
 
 dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has schema customization using the [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macro, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Troubleshooting](/docs/deploy/ci-jobs#troubleshooting).
 
+## Git providers who support CI
+
+If your git provider has a native dbt Cloud integration, you can set up continuous integration (CI) jobs within dbt Cloud.
```

Comment:
also wondering why we're not using the [_git-providers-supporting-ci.md](https://github.com/dbt-labs/docs.getdbt.com/pull/6692/files#diff-32c06828b06474ea82f2b1b980556a620e87ff1e7acd8ea0080076223569f3d2) snippet and hardcoding the table instead?